utils/constants.tsx

```diff
@@ -1612,6 +1612,7 @@ export const Constants = {
     } as Record<string, [string, number]>),
     CODE_PREVIEW_MAX_FILE_SIZE: 500000, // 500 KB
     HighlightedLanguages: {
+        1c: {name: '1C:Enterprise', extensions: ['bsl', 'os'], aliases: ['bsl']},
```

Comment:
You just have to wrap the field name in quotation marks if it starts with a number or contains anything that can't normally be the name of a variable.

```suggestion
        '1c': {name: '1C:Enterprise', extensions: ['bsl', 'os'], aliases: ['bsl']},
```